### PR TITLE
Fixbug #1039 : Chat screen UI

### PR DESCRIPTION
**Pull request by huynguyennovem**
1. The back button and the group name are not center-aligned

2. The avatar of a group is not correct

### CODE REVIEW

**Comment by EvGen94** [comment]
The limitation of 2 avatars was deliberately chosen, as 3 look too small, in the first screenshot here 3d avatar is cut

**Comment by huynguyennovem** [comment]
> The limitation of 2 avatars was deliberately chosen, as 3 look too small, in the first screenshot here 3d avatar is cut

Agreed, but need to confirm from design here. We can fix the image cutted issue, but it sill too small if it's 2 avatars in the stack

**Comment by huynguyennovem** [comment]
Updated (back to 2 avatars in stack)

File: lib/pages/chat/chat_header.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ChatHeader extends StatelessWidget {
               size: 38,
               imageUrl: isDirect ? avatars.first.link : icon,
               avatars: avatars,
-              stackSize: 26,
+              stackSize: 24,
```
Comment (EvGen94) [inline lib/pages/chat/chat_header.dart:41]: Why are we resizing here ?

**Review by EvGen94 — APPROVED** [review]

File: lib/pages/chat/chat_header.dart (hunk 1 above)
Comment (huynguyennovem) [reply lib/pages/chat/chat_header.dart:41]: 26 make it a little cut, so change to 24

**Pull request merged**